app/controllers/api/v1/keywords_controller.rb

```diff
@@ -38,10 +38,8 @@ def create_success_response
         }
       end
 
-      def source_name(_params)
-        # TODO: Read source_name from parameter
-
-        'Google'
+      def source_name(params)
```

Comment:
```suggestion
      def source_param
```

- methods that return the value of a param usually end with `_param`.
- as the `params` variable is an instance variable, then we do not need to pass it as an argument ;-)